slick.dataview.js

```diff
@@ -873,6 +875,11 @@
         onRowCountChanged.notify({previous: countBefore, current: rows.length}, null, self);
         onRowsChanged.notify({rows: diff}, null, self);
       }
+      if (filteredItems.length != previousFilteredItems.length) {
```

Comment:
keeping the "change" definition consistent with `onRowsChanged` (looking at length). Might want to do a tech debt later to fix this in both places.